floris/simulation/flow_field.py

```diff
@@ -197,14 +217,13 @@ def initialize_velocity_field(self, grid: Grid) -> None:
         self.v_sorted = self.v_initial_sorted.copy()
         self.w_sorted = self.w_initial_sorted.copy()
 
-        self.turbulence_intensity_field = self.turbulence_intensity * np.ones(
-            (
-                self.n_findex,
-                grid.n_turbines,
-                1,
-                1,
-            )
+        self.turbulence_intensity_field = self.turbulence_intensities[
+            :, np.newaxis, np.newaxis, np.newaxis
```

Comment:
We've used `None` instead of `np.newaxis` throughout the code, so I suggest to keep that consistent.

Reply:
Done!